analysers/analyser_merge_street_objects.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, config, logger = None):
 class SubAnalyser_Merge_Street_Objects(SubAnalyser_Merge_Dynamic):
     def __init__(self, config, error_file, logger, classs, level, otype, conflation, title, object, selectTags, generateTags, mapping, source, layer):
         SubAnalyser_Merge_Dynamic.__init__(self, config, error_file, logger)
-        self.def_class_missing_official(item = 8360, id = classs, level = level, tags = ['merge', 'leisure'],
+        self.def_class_missing_official(item = 8360, id = classs, level = level, tags = ['merge', 'leisure', 'fix:survey', 'fix:picture'],
```

Comment:
osmose produce a merge, so mapillary found it on a street level imagery. so imho fix:chair is also doable

Reply:
Same as above